runtime/vm/jfr.cpp

```diff
@@ -186,7 +186,7 @@ writeOutGlobalBuffer(J9VMThread *currentThread, bool finalWrite, bool dumpCalled
 	j9tty_printf(PORTLIB, "\n!!! writing global buffer %p of size %p\n", currentThread, vm->jfrBuffer.bufferSize - vm->jfrBuffer.bufferRemaining);
 #endif /* defined(DEBUG) */
 
-	if (areJFRBuffersReadyForWrite(currentThread)) {
+	if (vm->jfrState.isStarted && (NULL != vm->jfrBuffer.bufferCurrent)) {
```

Comment:
While you're here, could you please correct the typo on line 212: fluah -> flush.